foreshadow/tests/utils.py

```diff
@@ -40,6 +40,23 @@ def get_file_path(file_type, file_name):
     return os.path.join(test_path, file_type, file_name)
 
 
+def dynamic_import(attribute, module_path):
```

Comment:
might be worth changing this to use the registry once SerializerMixin is merged in.

Reply:
have some comments on the registry :)